Plugins/io.healthchecks/plugin-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "io.healthchecks",
+  "display_name": "Healthchecks",
+  "check_interval": 300,
+  "header_api_key": true
```

Comment:
I think this is from old code and should not be set; will investigate.

(Healthchecks apparently uses a different API key per “project”, so I made it user configurable per plug-in instance)

Reply:
Checked: indeed superfluous. Removed.